package.json

```diff
@@ -8,7 +8,13 @@
     "sass": "node ./scripts/sass.js",
     "build": "node ./scripts/build.js",
     "serve": "node ./scripts/serve.js",
-    "test": "./node_modules/.bin/mocha -R spec ./tests",
+    "test": "./node_modules/.bin/mocha -R spec ./tests && dropdb --if-exists epoch_test && createdb epoch_test && DATABASE_URL=postgres://localhost/epoch_test npm run db-migrate && npm run test-categories && npm run test-boards && npm run test-threads && npm run test-posts && npm run test-notifications && npm run test-users",
+    "test-notifications": "./node_modules/.bin/lab test/notifications.js --flat -v",
+    "test-categories": "./node_modules/.bin/lab test/categories.js --flat -v",
+    "test-boards": "./node_modules/.bin/lab test/boards.js --flat -v",
+    "test-threads": "./node_modules/.bin/lab test/threads.js --flat -v",
+    "test-posts": "./node_modules/.bin/lab test/posts.js --flat -v",
+    "test-users": "./node_modules/.bin/lab test/users.js --flat -v",
```

Comment:
Testing should be encompassed in 1 entry point. This is ok for now, but change in the future.